app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="home_page">
+  <div class="app-title">
+    <h2>Wedding  Cakes</h2>
+  </div>
+  <div class="log_in">
```

Comment:
- [ ] Kindly ensure you separate the login and signup pages to match the design given, here is what is expected of the pages

![image](https://user-images.githubusercontent.com/61408860/189099916-34b2c60f-a5c8-4df8-9f86-b8efa6fb49f9.png)

![image](https://user-images.githubusercontent.com/61408860/189100010-914fb653-00dd-4a86-87a0-6a9440d0430c.png)